projects/angular/src/accordion/stepper/stepper-panel.html

```diff
@@ -0,0 +1,64 @@
+<ng-container *ngIf="panel | async; let panel">
+  <div [ngClass]="getPanelStateClasses(panel)">
+    <div class="clr-accordion-header clr-stepper-header">
+      <button
+        type="button"
+        class="clr-accordion-header-button"
+        (click)="togglePanel()"
+        [id]="getAccordionHeaderId(panel.templateId)"
+        [disabled]="isAccordion && panel.disabled"
+        [attr.aria-disabled]="!isAccordion && panel.disabled"
+        [attr.aria-controls]="getAccordionContentId(panel.templateId)"
+        [attr.aria-expanded]="panel.open"
+        [class.clr-accordion-header-has-description]="(accordionDescription.changes | async)?.length || accordionDescription.length"
+        #headerButton
+      >
+        <span class="clr-step-status">
+          <cds-icon shape="angle" direction="right" class="clr-accordion-angle clr-step-angle"></cds-icon>
+          <cds-icon
+            status="danger"
+            *ngIf="panel.status === AccordionStatus.Error"
+            shape="exclamation-circle"
+            class="clr-step-error-icon"
+          ></cds-icon>
+          <cds-icon
+            status="success"
+            *ngIf="panel.status === AccordionStatus.Complete"
+            shape="check-circle"
+            class="clr-step-complete-icon"
+          ></cds-icon>
+        </span>
+        <span class="clr-step-title">
+          <span class="clr-step-number">{{panelNumber}}.</span>
+          <ng-content select="clr-accordion-title, clr-step-title"></ng-content>
+        </span>
+        <ng-content select="clr-accordion-description, clr-step-description"></ng-content>
+      </button>
+      <div class="clr-sr-only" role="status">
+        <ng-container *ngIf="panel.status === AccordionStatus.Error"> {{ stepErrorText(panelNumber)}} </ng-container>
```

Comment:
If the `AccordionStatus` enum isn't part of the public API, we should rename it to `StepperPanelStatus`; otherwise, we should create a Jira ticket to do that in v18 and reference it where the enum is declared.

Reply:
Okay, I suppose that's fine. However, the `AccordionStatus` property should be moved from the accordion panel class to the stepper panel class in this PR because it's no longer used in the accordion panel. See 92e1e8da636fdcf12165e54f87f84d014625adcd.

Reply:
That make sense.
Corrected in commit 93ae179